custom_model_runner/drum_server_api.yaml

```
@@ -22,6 +22,18 @@ paths:
                     description: Status message.
               example: 
                 message: OK
+        513:
+          description: 'Not healthy. Requires DRUM server has to be started with --with-error-server option'
```

Maybe provide a more descriptive explaination:
```suggestion
          description: 'Error loading/initializing model. Requires DRUM server has to be started with --with-error-server option'
```